pkg/render/logstorage.go

```diff
@@ -138,6 +138,39 @@ const (
 )
 
 const (
+	// ElasticsearchKeystoreSecret Currently only used when FIPS mode is enabled, we need to initialize the keystore with a password.
+	ElasticsearchKeystoreSecret         = "tigera-secure-elasticsearch-keystore"
+	ElasticsearchKeystoreEnvName        = "KEYSTORE_PASSWORD"
+	ElasticsearchKeystoreHashAnnotation = "hash.operator.tigera.io/keystore-password"
+
+	keystoreInitContainerName = "elastic-internal-init-jvm-keystore"
+	keystoreInitScript        = `#!/usr/bin/env bash
+set -eux
+
+echo "Initializing jdk keystore and setting it to BouncyCastleFipsProvider..."
+
+/usr/share/elasticsearch/jdk/bin/keytool \
+	-destkeystore /usr/share/elasticsearch/config/cacerts.bcfks \
+	-deststorepass ${%s} \
+	-deststoretype bcfks \
+	-importkeystore \
+	-providerclass org.bouncycastle.jcajce.provider.BouncyCastleFipsProvider \
+	-providerpath /usr/share/bc-fips/bc-fips.jar \
+	-srckeystore /usr/share/elasticsearch/jdk/lib/security/cacerts \
+	-srcstorepass changeit \
+	-srcstoretype jks
+
+echo "Keystore initialization successful."
+
+echo "Initializing Elasticsearch keystore..."
+
+bin/elasticsearch-keystore create -p <<EOF                                                                                               
```

Comment:
nit. we can use the absolute path `/usr/share/elasticsearch/bin/elasticsearch-keystore` in case the `WORKDIR` is changed in the future.